package-lock.json

```diff
@@ -97,22 +97,78 @@
       "integrity": "sha512-eqz8c/0kwNi/OEHQfvIuJVLTst3in0e7uTKeuY+WL/zfKn0xVujOTp42bS/vUUokhK5P2BppLd9JXMOMHcgbjA==",
       "dev": true
     },
+    "@ungap/promise-all-settled": {
```

Comment:
Please update the version as well

https://github.com/microsoft/app-store-vsts-extension/blob/fbaa81b7d198fa5156c44ca87bf332850d71b8a9/package-lock.json#L2-L4

Reply:
Thanks! Fixed